resources/documentation/multiple_icebergs.rst

```diff
@@ -136,38 +177,45 @@ To do so, edit again file /etc/bluebanquise/inventory/cluster/icebergs/iceberg1
 
   [iceberg1]
   management1
-  login1
-  c[001:004]
+  login[1:2]
+  storage[1:6]
 
-Note: it is possible to add ranges of nodes, like in this example c[001:004]. This is a different syntax than clustershell.
+.. note::
+  As you can see, it is possible to add ranges of nodes, like in this example
+  with login[1:2]. This is a different syntax than *nodeset*.
 
-Check again groups:
+Check groups again:
 
 .. code-block:: text
 
-  [root@mngt1-1 ~]# ansible-inventory --graph
+  [root@mngt1 ~]# ansible-inventory --graph
   @all:
   ...
     |--@iceberg1:
     |  |--management1
     |  |--login1
-    |  |--c001
-    |  |--c002
-    |  |--c003
-    |  |--c004
+    |  |--login2
+    |  |--storage1
+    |  |--storage2
+    |  |--storage3
+    |  |--storage4
+    |  |--storage5
+    |  |--storage6
     |--@ungrouped:
-  [root@mngt1-1 ~]#
+  [root@mngt1 ~]#
 
-And push again configuration using the default playbook for each host. It is possible to see what is going to be modified using --diff --check at ansible-playbook invocation.
+And push again configuration using the default playbook for each host.
```

Comment:
And push this new configuration using your dedicated playbook for each already 
deployed hosts.
?

Reply:
In theory, part of the cluster (maybe only mngt1) was already deployed with icebergs system deactivated. So I suggest here to re-apply conf in case of.